include/xtensor/xstrided_view.hpp

```diff
@@ -276,11 +276,15 @@ namespace xt
 
         void assign_temporary_impl(temporary_type&& tmp);
 
+        using base_type::set_offset;
+
         template <class C>
         friend class xstepper;
         friend class xview_semantic<self_type>;
         friend class xaccessible<self_type>;
         friend class xconst_accessible<self_type>;
+        template <class CT>
```

Comment:
This template parameter name should be different from that of the `xstrided_view` class to avoid shadowing (this should fix the CI).

Reply:
Changed from CT to D.